README.md

```diff
@@ -515,14 +515,64 @@ also emitted from the client via a `metrics` event:
 ```
 
 #### client.on('after', function(req, res, err) {... });
-StringClient and JsonClient emit an `after` event after each completed request
+The client will emit an `after` event after each completed request
 (successful or not). This can be useful for attaching after handlers to report
 metrics or other useful information about the client. When using this event,
 the timings and metrics information is available via the request getters,
-`req.getTimings()` and `req.getMetrics()`. Note that the low level HttpClient
-does not support this method, as that client is a thin wrapper over the http
-streams. As such, the user is in full control of the request lifecycle, and
-`after` would not have a clear meaning.
+`req.getTimings()` and `req.getMetrics()`.
+
+As the low level HttpClient's callback interface exposes the streams
```

Comment:
nit: Maybe separate this section by creating a `##### Notes on using the 'after' event with HttpClient instances`?

Also, it's not clear from the documentation when the `'after'` event fires for String and JSON clients. If I understand correctly, they fire after body parsing is done for StringClients and after JSON parsing is done for JSON clients. It might be worth it to describe that here?